backend/app/usecase/shortlink/updater_test.go

```diff
@@ -390,7 +390,9 @@ func TestShortLinkUpdaterPersist_UpdateShortLink(t *testing.T) {
 			assert.Equal(t, testCase.expectedShortLink.LongLink, shortLink.LongLink)
 			assert.Equal(t, testCase.expectedShortLink.Alias, shortLink.Alias)
 			assert.Equal(t, testCase.expectedShortLink.CreatedAt, shortLink.CreatedAt)
-			assert.Equal(t, true, shortLink.UpdatedAt.After(now))
+			if assert.NotEqual(t, false, shortLink.UpdatedAt != nil) {
```

Comment:
I tried to do `assert.NotEqual(t, nil, shortLink.UpdatedAt)` here, but I discovered that `NotEqual` will return `true` right away if the types are different, without checking the value. See https://godoc.org/github.com/stretchr/testify/assert#NotEqual

Reply:
I am thinking about adding a wrapper to `assert.NotNil` to app, which will properly check if it's not nil. We can also update all other assertions for not nil to use `assert.NotNil` instead. What do you think?

Sorry about adding this change last second. Just wanted to make sure nil memory access can't happen here for any of the test cases if they fail.

Reply:
@Coteh Let’s not add this one. It’s best practice to keep assertion library as simple as possible.

Reply:
I doubt that’s why the test fails. It was working correct for me before.